Formula/fstar.rb

```diff
@@ -37,16 +38,20 @@ def install
 
     system "opam", "config", "exec", "--", "make", "-C", "src", "boot-ocaml"
 
-    bin.install "bin/fstar.exe"
+    bin.install "bin/fstar.exe" => "_fstar.exe"
```

Comment:
Yep; I mean is there a reason to keep `_fstar.exe` in `bin/` if we already have `fstar.exe`? We could put it under e.g. `libexec/` instead.